extensions/httpserver/httpserver_hsminweb.lua

```diff
@@ -637,7 +637,7 @@ local webServerHandler = function(self, method, path, headers, body)
         if itBeCGI then
         -- do external script thing
 
--- this is a horrible horrible hack...
+-- this is a horrible hack...
```

Comment:
Happy to drop if you're attached to this. You can see that many duplications are unintentional/errant. I've left some duplications in place.

Alternatively,
```suggestion
-- this is a horrible, horrible, hack...
```